docs/getting_started/openstack.md

```diff
@@ -218,6 +218,16 @@ spec:
       insecureSkipVerify: true
 ```
 
-## Next steps
+# Security of node bootstrap
+
+As of kOps 1.27, the node bootstrap process uses an instance's UUID as a plaintext password.
```

Comment:
well I am thinking to make it at least little bit better: if I can somehow to verify that the request ip address is the one which the instance should have. And also if there is way to make better password algorithm

Reply:
Does Openstack prevent interception/spoofing of IP within the network like Amazon VPC does?

Would it be possible to pass a per-instance credential through userdata or vendordata? The `Instance` task could mint a specific, time-limited credential if it had a way to pass it to the instance reasonably securely.

Reply:
@johngmyers yep there is thing called "portsecurity", you cannot fake your ip address.

Is the credential needed only one time? I am just thinking is it one possible way to check is instance < 1 hour old in verifier? After that it will not accept requests